app/src/main/java/com/google/android/samples/socialite/ui/player/preloadmanager/PreloadManagerWrapper.kt

```diff
@@ -160,7 +160,7 @@ private constructor(
     class PreloadStatusControl : TargetPreloadStatusControl<Int> {
         override fun getTargetPreloadStatus(rankingData: Int): DefaultPreloadManager.Status {
             // By default preload first 3 seconds of the video
-            return DefaultPreloadManager.Status(STAGE_LOADED_TO_POSITION_MS, 3000L)
+            return DefaultPreloadManager.Status(STAGE_LOADED_FOR_DURATION_MS, 3000L)
```

Comment:
@MayuriKhinvasara can you comment on whether this is the right update after updating to the latest media3 release?

Reply:
There are a couple of changes to PreloadManager when updating to media3 1.5. We would have to test them before commits.